runtime/jcl/common/reflecthelp.c

```diff
@@ -1275,6 +1275,75 @@ Java_java_lang_reflect_Array_multiNewArrayImpl(JNIEnv *env, jclass unusedClass,
 	return result;
 }
 
+#if JAVA_SPEC_VERSION > 8
+/* The caller must have VM access. */
+J9ROMFieldShape *
+getROMFieldHelper(J9VMThread *vmThread, jclass declaringClass, jstring fieldName, UDATA *offsetResult)
+{
+	J9InternalVMFunctions *vmFuncs = vmThread->javaVM->internalVMFunctions;
+	J9Class *clazz = NULL;
+	J9ROMClass *romClass = NULL;
+	J9ROMFieldShape *romField = NULL;
+	j9object_t fieldNameObj = NULL;
+	J9ROMFieldWalkState state;
+
+	Assert_JCL_mustHaveVMAccess(vmThread);
+
+	if ((NULL == fieldName) || (NULL == declaringClass)) {
+		goto nullpointer;
+	}
+	fieldNameObj = J9_JNI_UNWRAP_REFERENCE(fieldName);
+	if (NULL == fieldNameObj) {
+		goto nullpointer;
+	}
+	clazz = J9VM_J9CLASS_FROM_JCLASS(vmThread, declaringClass);
+	Assert_JCL_notNull(clazz);
+	romClass = clazz->romClass;
+
+	/* primitives/arrays don't have fields */
+	if (J9ROMCLASS_IS_PRIMITIVE_OR_ARRAY(romClass)) {
+		return NULL;
+	}
+
+	/* loop over the fields */
+	memset(&state, 0, sizeof(state));
+	romField = romFieldsStartDo(romClass, &state);
+	for (romField = romFieldsStartDo(romClass, &state);
+		NULL != romField;
+		romField = romFieldsNextDo(&state)
+	) {
+		J9UTF8 *romFieldName = J9ROMFIELDSHAPE_NAME(romField);
+
+		/* compare name with this field */
+		if (0 != vmFuncs->compareStringToUTF8(vmThread, fieldNameObj, 0, J9UTF8_DATA(romFieldName),
+			J9UTF8_LENGTH(romFieldName))
+		) {
```

Comment:
When the arguments don't all fit on the line, our coding standard says that none should be on the first line.
Continued lines should be indented two tabs relative to the first line.